calculator.sh

```
@@ -3,7 +3,7 @@
 # Simple Calculator Script
 
 echo "Enter first number:"
-read num1
+read num4
```

Shellcheck (0.11.0)

[info] read without -r will mangle backslashes.

(SC2162)